mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -315,6 +316,19 @@ impl UtxoRpcError {
 
         false
     }
+
+    pub fn is_response_too_large(&self) -> bool {
+        if let UtxoRpcError::ResponseParseError(ref json_err) = self {
+            if let JsonRpcErrorType::Response(_, json) = &json_err.error {
+                return json["code"] == RESPONSE_TOO_LARGE_CODE
+                    || json["message"]
+                        .as_str()
+                        .unwrap_or_default()
+                        .contains("response too large");
```

Comment:
Do we need to check first the `code` then the `message` here? What other codes return `response too large` message? and shouldn't the message be enough in the case that there are multiple codes returned?

Reply:
`code: -32600` is the only error code returned for `response too large` from electrum and yes, checking for only the message doesn't hurt too!

Reply:
> `code: -32600` is the only error code returned for `response too large` from electrum and yes, checking for only the message doesn't hurt too!

In this case we should check only for the code I guess

Reply:
Just another note, this is different from `is_tx_not_found_error` since the error in this case is returned from the coin daemon not from the electrum server itself